terraform/connect.gov.tf

```diff
@@ -40,18 +40,12 @@ resource "aws_route53_record" "connect_gov_connect_gov_mx" {
   records = ["1 aspmx.l.google.com", "5 alt2.aspmx.l.google.com", "5 alt1.aspmx.l.google.com", "10 alt3.aspmx.l.google.com", "10 alt4.aspmx.l.google.com"]
 }
 
-resource "aws_route53_record" "connect_gov_connect_gov_txt" {
-  zone_id = "${aws_route53_zone.connect_gov_zone.zone_id}"
-  name    = "connect.gov"
-  type    = "TXT"
-  ttl     = 1800
-  records = ["v=spf1 ~all", "google-site-verification=j3qyXzcDt_O3t0sdYy6FCQlYJnV5ASd0GYIhicPPzOg"]
-}
+module "connect_gov__email_security" {
+  source = "./email_security"
 
-resource "aws_route53_record" "connect_gov__dmarc_connect_gov_txt" {
   zone_id = "${aws_route53_zone.connect_gov_zone.zone_id}"
-  name    = "_dmarc.connect.gov"
-  ttl     = "900"
-  type    = "TXT"
-  records = ["${local.dmarc_reject}"]
+  txt_records = [
+    "v=spf1 ~all",
```

Comment:
I assume this should actually be a deny (`-all`), but didn't want to change any behavior in this pull request.